### ISSUE TITLE
Please invite me to the GitHub Community Organization

### ISSUE THREAD

**Issue by Royal0106** [EddieHubCommunity/support#8955]
### Name

Justin Cromer

### Discord Username (if applicable)

dt.eth

### Additional Context

I actually first heard about this community through my good friend Eddie. We met a while back when we were both working on our own side projects, trying to build out web apps and get them off the ground. Eddie had joined this group and was telling me all about the amazing support and camaraderie he was finding here.

He said it was just such a welcoming, collaborative environment, where everyone was eager to share their expertise, bounce ideas off each other, and really help each other grow as developers. And the diversity of skills and backgrounds represented in the community was really impressive - there were people at all different stages of their careers, from seasoned veterans to total newbies like me at the time.

So in short, that's why I'm so eager to get more involved with this community.
